README.md

```diff
@@ -258,7 +247,7 @@ indexData = {
     }
 };
 
-client.index(indexData, autoCreateFields = true).then((resp) => {
+client.insert(insertData, autoCreateColumns = true).then((resp) => {
```

Comment:
"autoCreateColumns" parameter should be removed.
Check this PR: https://github.com/SlicingDice/slicingdice-javascript/pull/12